ihatemoney/utils.py

```diff
@@ -12,6 +12,31 @@
 from datetime import datetime, timedelta
 
 import csv
+import requests
+
+
+class CurrencyConverter(object):
+    api_url = "https://api.exchangerate-api.com/v4/latest/USD"
+    response = []
+
+    def __init__(self):
+        self.response = requests.get(self.api_url).json()
+
+    def get_currencies(self):
+        currencies = []
+        for rate in self.response["rates"]:
+            currencies.append((rate, rate))
+        return currencies
+
+    def exchange_currency(self, amount, currency1, currency2):
+        if currency1 == currency2:
+            return amount
+
+        conversion_rate1 = self.response["rates"][currency1]
+        conversion_rate2 = self.response["rates"][currency2]
```

Comment:
the logic seems weird to me : should we issue a call to the API with the correct currency instead, to just get the currency rate and apply it?